src/System.ServiceProcess.ServiceController/src/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <PackageConfigurations>
       netstandard;
+      netstandard-Windows_NT;
```

Comment:
Why is this needed? In general I like to avoid RID netstandard configurations.

Reply:
I agree, but as explained above, this is required because we are harvesting a windows specific netstandard1.3 implementation for this package, which will be prefered over the lib/netstandard2.0 implementation for UAP.

Reply:
@weshaggard why do you want to avoid RID-netstandard configurations?  What if we want to share a windows implementation between netcoreapp and UAP?  That's the reason why those old harvested assets existed that were RID-specific and targeted netstandard.

Reply:
There is nothing impure about having a RID + netstandard assets.  Consider that the netstandard version targeted by the assembly represents the managed API needed by that assembly and the RID represents the platform API/behavior.  It's actually more pure to have the assembly be win+netstandard since that's technically all it depends on.  Making it win+UAP / win+netcoreapp implies it depends on something specific to those TFMs which at least the package asset does not.

Reply:
By purity I'm thinking about netstandard being pure IL and no pinvokes, which is where I would like most libraries to live. I know that isn't a reality but it is my ideal world. I agree using a combination of netstandard+RID can be useful in some scenarios but it also adds some risk as the native dependencies may or may not work on any netstandard supported platform. I guess that is true for all assets that go under a RID and I've never really liked how "undefined" RID's are which also leads me to not liking these. At any rate netstandard+RID is a tool and we should use it if necessary but that doesn't make me like it :)